react-app/ccf/src/components/faq/FAQComponent.css

```
@@ -37,33 +37,41 @@
-    flex: 1;
+    flex: 1 1 0;
+    min-width: 0;
     gap: 12px;
 }
 
 .faq-question-text {
-    flex: 1;
+    flex: 1 1 0;
     min-width: 0;
+    overflow-wrap: anywhere;
+    word-break: break-word;
```

Stylelint (17.11.1)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)